CommonTools/PileupAlgos/interface/PuppiAlgo.h

```diff
@@ -4,12 +4,14 @@
 #include "FWCore/Framework/interface/Frameworkfwd.h"
 #include "FWCore/ParameterSet/interface/ParameterSet.h"
 #include "CommonTools/PileupAlgos/interface/PuppiCandidate.h"
+#include "FWCore/ParameterSet/interface/ConfigurationDescriptions.h"
```

Comment:
The correct include here is
```
#include "FWCore/ParameterSet/interface/ParameterSetDescription.h"
```
(possibly added right below the include of the ParameterSet.h)

Reply:
@alefisico please implement also this one.
Then we will be probably ready to go.